resources/views/env.blade.php

```diff
@@ -58,6 +58,14 @@
 # Polr will redirect logged off users to this URL
 SETTING_INDEX_REDIRECT={{$ST_INDEX_REDIRECT}}
```

Comment:
Hi @cydrobolt, thanks for the review.
First, I thought it would be necessary to separate the redirect url for hidden interface or when short url not exist. But finally I considered that I still can use `SETTING_INDEX_REDIRECT` setting.
I had fix the code.

Regards.